distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP/Response.enso

```diff
@@ -57,8 +55,8 @@ type Response
              example_headers = Examples.get_response.headers
     headers : Vector
     headers self =
-        header_entries = Vector.from_polyglot_array (Http_Utils.get_headers self.internal_http_response.headers)
-        header_entries.map e-> Header.new e.getKey e.getValue
+        header_map = self.internal_http_response.headers.map
+        header_map.keys.map k-> Header.new k (header_map.get k)
```

Comment:
```suggestion
        header_map.to_vector.map p-> Header.new p.first p.second
```

IMHO this is preferred because we traverse the map once and do not need to run `get` so much.

However since this is not a performance sensitive operation, this doesn't matter much, so if you think the current form is more readable, I think it's ok to keep unchanged.